tck/src/main/java/org/eclipse/microprofile/lra/tck/TckLRATypeTests.java

```diff
@@ -215,6 +216,30 @@ public void neverWithEndRA() {
     public void neverWithoutEndLRA() {
         resourceRequest(NEVER_WITH_END_FALSE_PATH, false, 200, MethodLRACheck.NOT_PRESENT, false);
     }
+    
+    @Test
+    public void requiredWithClosedLRA() {
+        String closedLRA = resourceRequest(REQUIRED_PATH, false, 200, MethodLRACheck.NONE, false);
```

Comment:
I think it might aid readability if this was:
> String lraId = resourceRequest(....
Of course, I might be wrong about what you are expecting here but I think it is an ID?